benchmarks/maxtext_xpk_runner.py

```diff
@@ -40,6 +40,15 @@
 # Assumes you have xpk installed in a git clone repo of ~/{wl_config.xpk_path}/xpk.py
 _DEFAULT_MAXTEXT_BASE_DOCKER_IMAGE_NAME = 'maxtext_base_image'
 
+# The minimum set of tuning params required for pathways.
+BASE_PATHWAYS_TUNING_PARAMS = {
+    'checkpoint_storage_use_ocdbt': False,
```

Comment:
Curious to know why these params are needed for Pathways only - checkpoint_storage_use_ocdbt and checkpoint_storage_use_zarr3 .

Reply:
We don't currently support these storage types at the moment for Pathways. Support will come eventually, but until then we'll skip it.